ui/tests/acceptance/enterprise-kmip-test.js

```diff
@@ -210,7 +210,7 @@ module('Acceptance | Enterprise | KMIP secrets', function (hooks) {
 
   test('it can create a role', async function (assert) {
     // moving create scope here to help with flaky test
-    const path = await mount();
+    const path = await mount(this);
```

Comment:
I'm not sure if this makes a difference, but I noticed we invoke mount with this in the [`create a scope` test](https://github.com/hashicorp/vault/blob/bd23a60c99e35dbc9d77d6cadef89bb3315cfdcd/ui/tests/acceptance/enterprise-kmip-test.js#L172)

Reply:
The mount function only accepts a boolean attribute, so I'm not sure we should pass `this` in either of them 🤔

Reply:
Yeah I wasn't sure why that was added. I'll remove it from both